requirements-osbs.txt

```diff
@@ -141,10 +141,10 @@ zope.event==4.5.0
 zope.interface==5.4.0
 
 # Build dependencies
-Cython==3.0a9
+Cython==0.29.23
```

Comment:
Seems weird to move off the prerelease train in this PR.

Reply:
Must've changed it by accident. . Changed it back to `Cython==3.0a9`